.env.txt

```
@@ -0,0 +1 @@
+OPENAI_API_KEY=sk-bc9a3a95e190fc9cf5525a4726b16658
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)